autopush/utils.py

```diff
@@ -265,6 +265,9 @@ class WebPushNotification(object):
     # Whether this notification should follow legacy non-topic rules
     legacy = attrib(default=False)  # type: bool
 
+    # the version of the VAPID header
+    vapid_version = attrib(default='draft01')  # type: str
```

Comment:
This field is never filled in (will always be draft01) -- arguably it's not the best place for it either.

Since it's only needed by the webpush handler method, I'd just have that method take it as an explicit kwarg, it's already in its **kwargs  as part of the WebPushRequestSchema